src/site/apt/index.apt.vm

```diff
@@ -44,8 +44,8 @@ ${project.name}
    * {{{./dependency-convergence-mojo.html}project-info-reports:dependency-convergence}} is used to generate the
    Project Dependency Convergence report for (reactor) builds.
 
-   * {{{./dependency-info-mojo.html}project-info-reports:dependency-info}} is used to generate code snippets to
-   be added to build tools.
+   * {{{./dependency-info-mojo.html}project-info-reports:dependency-info}} is used to generate code snippets
```

Comment:
is used to generate --> generates

Reply:
This goes throughout all bulletpoints. I'd leave as is and do this in another PR.